src/pybind/mgr/cephadm/ssl_cert_utils.py

```diff
@@ -67,39 +59,42 @@ def generate_root_cert(self, addr: str) -> Tuple[str, str]:
 
         return (cert_str, key_str)
 
-    def generate_cert(self, host: str, addr: str) -> Tuple[str, str]:
-        have_ip = True
+    def generate_cert(self, hosts: Any, addrs: Any) -> Tuple[str, str]:
```

Comment:
it seems like we really want these to either be strings or a list of strings. We should probably make that explicit in the typing to get more out of mypy rather than just setting this to `Any`

Reply:
Fixed